assets/src/blocks/SocialMedia/SocialMediaEmbed.js

```diff
@@ -1,42 +1,44 @@
 import { OEMBED_EMBED_TYPE, FACEBOOK_EMBED_TYPE } from './SocialMediaConstants.js';
 
-export const SocialMediaEmbed = ({
+export const SocialMediaEmbed = ( {
   alignmentClass,
   embedCode,
   facebookPageTab,
   facebookPageUrl,
   embedType,
-}) => {
+} ) => {
   if (
-    (embedType === OEMBED_EMBED_TYPE && !embedCode) ||
-    (embedType === FACEBOOK_EMBED_TYPE && !facebookPageUrl)
+    ( embedType === OEMBED_EMBED_TYPE && ! embedCode ) ||
+    ( embedType === FACEBOOK_EMBED_TYPE && ! facebookPageUrl )
   ) {
     return null;
   }
 
   return (
-    <div className={`social-media-embed ${alignmentClass ?? ''}`}>
-      {(embedType === OEMBED_EMBED_TYPE && embedCode) ?
-        <div dangerouslySetInnerHTML={{ __html: embedCode }} /> :
-        <>
-          <iframe
-            className="social-media-embed-facebook social-media-embed-facebook--small"
-            src={`https://www.facebook.com/plugins/page.php?href=${encodeURIComponent(facebookPageUrl)}&tabs=${facebookPageTab}&width=240&height=500&small_header=false&adapt_container_width=true&hide_cover=false&show_facepile=true`}
-            scrolling="no"
-            frameBorder="0"
-            allowtransparency="true"
-            allow="encrypted-media"
-          />
-          <iframe
-            className="social-media-embed-facebook social-media-embed-facebook--large"
-            src={`https://www.facebook.com/plugins/page.php?href=${encodeURIComponent(facebookPageUrl)}&tabs=${facebookPageTab}&width=500&height=500&small_header=false&adapt_container_width=true&hide_cover=false&show_facepile=true`}
-            scrolling="no"
-            frameBorder="0"
-            allowtransparency="true"
-            allow="encrypted-media"
-          />
-        </>
+    <div className={ `social-media-embed ${ alignmentClass ?? '' }` }>
+      { ( embedType === OEMBED_EMBED_TYPE && embedCode )
+        ? <div dangerouslySetInnerHTML={ { __html: embedCode } } />
+        : <>
+					<iframe
+					  className="social-media-embed-facebook social-media-embed-facebook--small"
+					  src={ `https://www.facebook.com/plugins/page.php?href=${ encodeURIComponent( facebookPageUrl ) }&tabs=${ facebookPageTab }&width=240&height=500&small_header=false&adapt_container_width=true&hide_cover=false&show_facepile=true` }
+					  scrolling="no"
+					  frameBorder="0"
+					  allowTransparency="true"
```

Comment:
This change causes a console warning:
```
Warning: React does not recognize the `allowTransparency` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `allowtransparency` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

Reply:
I was not sure either what to do with it, I may just remove it as it does not seem to serve any purpose.

Reply:
Done